graylog2-server/src/main/java/org/graylog/plugins/views/search/engine/suggestions/SuggestionResponse.java

```diff
@@ -35,12 +35,14 @@
 
     public abstract Optional<SuggestionError> suggestionError();
 
+    public abstract long sumOtherDocsCount();
 
-    public static SuggestionResponse forSuggestions(final String field, final String input, final List<SuggestionEntry> suggestions) {
-        return new AutoValue_SuggestionResponse(field, input, suggestions, Optional.empty());
+
+    public static SuggestionResponse forSuggestions(final String field, final String input, final List<SuggestionEntry> suggestions, long sumOtherDocsCount) {
+        return new AutoValue_SuggestionResponse(field, input, suggestions, Optional.empty(), sumOtherDocsCount);
     }
 
     public static SuggestionResponse forError(final String field, final String input, final SuggestionError error) {
-        return new AutoValue_SuggestionResponse(field, input, Collections.emptyList(), Optional.of(error));
+        return new AutoValue_SuggestionResponse(field, input, Collections.emptyList(), Optional.of(error), -1);
```

Comment:
We could also use a boxed type (`Long`) for the field and return set it to `null`. This way we prevent a consumer to erroneously use the number from this field although it's an error. If we want to the response cleaner, we could even annotate the class with `@JsonInclude(JsonInclude.Include.NON_NULL)` to prevent the field from being serialized in that case.

Reply:
Good point, thanks, implemented it like that!